Robot_Adapter/CRUD/Create/Loads/LoadCombination.cs

```diff
@@ -41,9 +43,26 @@ private bool CreateCollection(IEnumerable<LoadCombination> lComabinations)
                 if (!CheckNotNull(lComb))
                     continue;
 
-                if (m_RobotApplication.Project.Structure.Cases.Exist(lComb.Number)!=-1)
+                IRobotCombinationType robotCombinationType = IRobotCombinationType.I_CBT_ULS;
+
+                //Assign IRobotCombinationType based on the LoadCombinationType fragment
+                var combTypeFragment = lComb.FindFragment<LoadCombinationType>();
+                if (combTypeFragment == null)
+                {
+                    BH.Engine.Base.Compute.RecordWarning($"LoadCombination with Number {lComb.Number} does not have a LoadCombinationType fragment. Defaulting to ULS.");
```

Comment:
```suggestion
                    BH.Engine.Base.Compute.RecordWarning($"LoadCombination with Number {lComb.Number} does not have a LoadCombinationType fragment. Defaulting to Eurocode ULS.");
```
Not sure how Robot handles design codes but may be worth including that it's EC.